packages/client/src/pdfRenderer/index.ts

```diff
@@ -64,7 +64,7 @@ export function createPDF(
   return pdfMake.createPdf(
     JSON.parse(definitionString),
     undefined,
-    template.fonts[intl.locale] || template.fonts[intl.defaultLocale]
+    template.fonts
```

Comment:
@Zangetsu101 by removing this, are we not removing the possibility to support say a cert in Bengali and a cert in English?  The font file may not contain all characters.

Reply:
Not necessarily as the certificateConfiguration we are getting from country config isn't restricted to be only a single font definition so we can just pass a Bengali font & a English one and use them both on the certificate or a single one if we want.So I decided to flatten the structure or am I missing something?

Reply:
In the future if we perhaps render a different certificate depending on the selected language, this configuration would still support that as which font needs to be used is mentioned in the svg & where fetch that font from would be defined in this configuration